internal/provider/resources/resource_deployment_test.go

```diff
@@ -282,7 +283,7 @@ func TestAcc_ResourceDeploymentStandard(t *testing.T) {
 				ResourceName:            azureCeleryResourceVar,
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"external_ips", "oidc_issuer_url", "environment_variables.1.value"}, // environment_variables.0.value is a secret value
+				ImportStateVerifyIgnore: []string{"external_ips", "oidc_issuer_url", "scaling_status", "environment_variables.1.value"}, // environment_variables.0.value is a secret value
```

Comment:
Sanity check: `ImportStateVerifyIgnore` means that TF won't return this in the output of `terraform plan` or `terraform apply`? How do these affect the assertions?

Reply:
It means that for this test case, we are just ignore those values after an import - we will not check if they are equivalent
Since the first 3 values are computed, it doesnt really matter that the values are not the same and the last value cannot be the same since its a secret
I had it this way since it was throwing errors for those 3